Fuzzlyn/Execution/ProgramExecutor.cs

```diff
@@ -73,19 +73,20 @@ ProgramResult RunAndGetResult(byte[] bytes)
         // Launches a new instance of Fuzzlyn to run the specified programs in.
         public static List<ProgramPairResults> RunSeparately(List<ProgramPair> programs)
         {
-            string dotnet;
+            string host;
             using (Process proc = Process.GetCurrentProcess())
             using (ProcessModule mm = proc.MainModule)
             {
-                dotnet = mm.FileName;
+                host = mm.FileName;
             }
 
             string fuzzlyn = Assembly.GetExecutingAssembly().Location;
             string fuzzlynDir = Path.GetDirectoryName(fuzzlyn);
+            bool hostIsFuzzlyn = host.Equals("fuzzlyn", StringComparison.OrdinalIgnoreCase);
             ProcessStartInfo info = new ProcessStartInfo
             {
-                FileName = dotnet,
-                Arguments = $"\"{fuzzlyn}\" --execute-programs",
+                FileName = host,
+                Arguments = hostIsFuzzlyn ? " --execute-programs" : "fuzzlyn --execute-programs",
```

Comment:
First one can be "--execute-programs" (no space) and the second one should be `"\"{fuzzlyn}\" --execute-programs"`.

Reply:
Fixed.